tox.ini

```diff
@@ -27,3 +27,9 @@ deps =
     check-manifest
 commands =
     check-manifest
+
+[testenv:bandit]
+deps =
+    bandit
```

Comment:
To be able to run this more easily, could we add bandit to the `requirements-test.txt` as well? This file is meant to include anything needed to test the project.

https://github.com/browniebroke/deezer-python/blob/master/requirements-test.txt

Reply:
Done